mod_reforged/hooks/config/character.nut

```diff
@@ -3,6 +3,11 @@
 ::Const.CharacterProperties.NegativeMoraleCheckBravery <- array(::Const.MoraleCheckType.len(), 0);
 ::Const.CharacterProperties.NegativeMoraleCheckBraveryMult <- array(::Const.MoraleCheckType.len(), 1.0);
 
+::Const.CharacterProperties.getFatigueRecovery <- function()
```

Comment:
This seems like a useful function and should be added via MSU imo. For now we can keep it in Reforged but call it `getFatigueRecoveryRate`